doc/design/validation.md

```diff
@@ -0,0 +1,25 @@
+# Requirements for being a valid DANDI asset
+
+## General
+1. All assets must have a non-zero file size.
+
+## NWB files
+- subject id
```

Comment:
```suggestion
- subject id (nwb: NWBFile.subject.subject_id)
```

Reply:
I think @bendichter 's idea to add explicit nwb schema paths is a nice one.  But I see only that @satra got them resolved without comments/changes.   Was that intentional?

Reply:
no i think i exposed a weird github bug. i added them to batch, but something refreshed and i hit refresh and my batch disappeared.

Reply:
closing this - they have been merged into a commit. so not a bug.